src/test/scala/org/ergoplatform/serialization/SerializationTests.scala

```diff
@@ -13,6 +17,9 @@ class SerializationTests extends ErgoPropertyTest with scorex.testkit.Serializat
     forAll(invalidHeaderGen) { b: Header =>
       val recovered = serializer.parseBytes(serializer.bytesWithoutInterlinks(b)).get.copy(interlinks = b.interlinks)
       recovered shouldBe b
+
+      val size = serializer.bytesWithoutInterlinks(b).length
+      recovered.size shouldBe size
```

Comment:
use `checkSize` and `checkSerializationRoundtrip`?

Reply:
@catena2w this one test case for non standard serialisation, since checkSerializationRoundtrip is defined in Scorex we can use this construction only for testing `parseBytes` and `toBytes`.